api/DbConn.php

```diff
@@ -0,0 +1,30 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: drazen.popov
+ * Date: 9/14/2015
+ * Time: 12:59 PM
+ */
+class dbConn
+{
+    private $dbName = 'dt_picker';
+    private $dbPass = '';
+    private $dbUser = 'root';
+    private $dbHost = 'localhost';
+    public $dbConn;
+
+    function connect() {
+        try {
+            $this->dbConn = new PDO('mysql:host=' . $this->dbHost . ';dbname=' . $this->dbName, $this->dbUser, $this->dbPass);
+            $this->dbConn->setAttribute(PDO::ATTR_ERRMODE, PDO::ERRMODE_EXCEPTION);
+            return $this->dbConn;
+
+        } catch (PDOException $e) {
+            $response=new Response(false, $e->getMessage(),false);
```

Comment:
According to single responsibility principle, every class should be responsible for one and only thing, in the whole system. Having the Response object here breaks that principle, and also mixes concepts from different layers of abstraction which should be avoided at all cost.( Response is the concept that deals with HTTP details and DbConn should not know about it )